set.mm

```diff
@@ -319234,7 +319268,7 @@ can be the countable union of rare closed subsets (where rare means
        is a complete metric space.  Remark: the assumption that the Banach
        space must be a (subcomplex) pre-Hilbert space is required because the
        definition of ` ClSubSp ` is based on an inner product.  If ` ClSubSp `
-       was generalized for arbitrary topological spaces, this assuption could
+       was generalized for arbitrary topological spaces, this assumption could
```

Comment:
topological modules

Reply:
maybe we can write "... generalized for arbitrary topological spaces/modules, ...".

Reply:
I was thinking of defining "closed linear subspace", and that is naturally defined for topological modules.  If you think that defining "closed subspace" has enough applications, then so be it (and indeed, that would naturally be defined for topological spaces). Then, the set of closed linear subspaces would be defined as the intersection of the sets of closed topological subspaces and of linear subspaces.  There is a (vague) policy in set.mm to be frugal on definitions, and add them only when needed, so I do not know if we should have both definitions.

(As for the grammar, I think it is "generalized TO".)